variants/aws-ecs-1-nvidia/Cargo.toml

```diff
@@ -30,6 +30,7 @@ included-packages = [
 # NVIDIA support
     "ecs-gpu-init",
     "nvidia-container-toolkit",
+    "nvidia-container-toolkit-ecs",
```

Comment:
nit: I prefer to list only the most specific set of leaf packages here - you could drop "nvidia-container-toolkit" everywhere since the `-k8s` or `-ecs` subpackage will pull that in by way of dependencies.

Reply:
Done :)
Thanks for the review!